src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -104,11 +104,11 @@ internal TokenCache UserTokenCache
             }
         }
 
-        /// <summary>
+/*        /// <summary>
```

Comment:
So you're hiding it by commenting it out? Can we just remove it? If we get feedback from developers to add it back, we can always find it in git commits and copy it back :)